test/implementation/util/fim_exceptions.clj

```diff
@@ -48,4 +48,21 @@
   (let [lines (friendly-exception-lines (Error. "message") ">>>")]
     (first lines) => #"Error.*message"
     (re-find #"^>>>" (first lines)) => falsey
-    (count (map #(re-find #">>>implementation.util.t_exceptions" %) (rest lines))) => (count (rest lines))))
+    (count (remove nil? (map #(re-find #">>>implementation.util.fim_exceptions" %) (rest lines)))) => (count (rest lines))))
```

Comment:
I suspect that the filename changed and it broke this test because
```
(map #(re-find #">>>implementation.util.t_exceptions" %) (rest lines))
```
was resulting in 
```
(nil nil nil nil)
```